### ISSUE TITLE
is there a way to temporarily disable CORS for testing?

### ISSUE THREAD

**Issue by christophertjones123** [cloudspout/cloudspout-button-panel#92]
I would like to disable CORS for testing purposes temporarily. Is that currently available? the way the README reads is that CORS is only enabled for basic auth but it seems like even if you are doing any kind of auth it will try and use CORS.

**Comment by fercasjr** [comment]
yes. there is a bug at least on the Grafana released version. you can fix it, but Grafana won't let you run it because of the signature so you will need enable debug mode and allow to run plugins without signatures.
